lua/neogen/generator.lua

```diff
@@ -0,0 +1,229 @@
+local ok, ts_utils = pcall(require, "nvim-treesitter.ts_utils")
+assert(ok, "neogen requires nvim-treesitter to operate :(")
```

Comment:
This should be moved to init.lua, as we want to warn users as soon as they install Neogen. Don't you think ?
Futhermore, I think the assert here is not good. We should use `notify`, with `ERROR`.

Reply:
require at init.lua can't delay load. I don't think users like the warning msg even though they don't use neogen right now.

Reply:
Right, okay to keep it there. What about using notify then instead of `assert` ?

Reply:
Yes, it is better than throwing the error make users feel panic.